lib/ansible/galaxy/collection/__init__.py

```diff
@@ -339,12 +339,12 @@ def verify_local_collection(local_collection, remote_collection, artifacts_manag
     ]
 
     # Find any paths not in the FILES.json
-    for root, dirs, files in os.walk(b_collection_path):
-        for name in files:
+    for root, dirs, filenames in os.walk(b_collection_path):
+        for name in filenames:
             full_path = os.path.join(root, name)
             path = to_text(full_path[len(b_collection_path) + 1::], errors='surrogate_or_strict')
             if any(fnmatch.fnmatch(full_path, b_pattern) for b_pattern in b_ignore_patterns):
-                display.v("Ignoring verification for %s" % full_path)
+                display.v("Ignoring verification for %s" % to_text(full_path))
```

Comment:
Why not go for an f-string then?
```suggestion
                display.v(f"Ignoring verification for {to_text(full_path)!s}")
```

Reply:
Largely because I was making as minimal changes as necessary

Reply:
and at that point you don't need `to_text` at all as `!s` or `!r` would do the work

Reply:
Ack. Althought, `to_text()` isn't exactly equivalent to `!s` / `!r`, FWIW. I agree that smaller surgical changes are better. And a bigger refactoring would deserve a separate PR.